web/beacon-app/src/config/sentry.ts

```diff
@@ -0,0 +1,26 @@
+import * as Sentry from '@sentry/react';
```

Comment:
This is the current Sentry configuration for rotational.app. I wanted to make sure we didn't forget to add this, but this may need to be updated.

Reply:
would you mind moving this to `application/config` instead ?